apps/test-blinky/src/main.cpp

```diff
@@ -4,17 +4,19 @@
  */
```

Comment:
The 2nd part of the training challenge specifies to create a tutorial_servo_control_can app to control the servo with a value from a CAN message. Please add this part.